.github/workflows/tlaplus.yml

```diff
@@ -0,0 +1,15 @@
+name: "TLA+ Spec Testing"
+
+on: [push, pull_request]
+
+jobs:
+
+  model-checker:
+    name: Run TLC Model Checker Against CCF Raft Spec
+    runs-on: ubuntu-latest
```

Comment:
Maybe we could use a larger runner: https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners#machine-specs-for-larger-runners

Reply:
It is also fairly easy to hook in a spare but beefy machine from the lab if budget is an issue.  For example, the TLA+ performance tests use a bare metal instance, and it's a plain Ubuntu with just the GitHub runner installed.

Reply:
Let's stick with the basic machine for now to get this merged and we can scale up later as we extend the CI